tests/Hedgehog.Tests/TreeTests.fs

```diff
@@ -0,0 +1,79 @@
+﻿module Hedgehog.Tests.TreeTests
+
+open Hedgehog
+open Swensen.Unquote
+open Xunit
+
+[<Fact>]
+let ``render tree with depth 0`` () =
+    Property.check <| property {
```

Comment:
We can have those run just once instead of 100 times. I'll leave them as-is for now as I'm planning to do some separate houskeeping to the whole test-suite.